pipeline/stop_places/sql/stop_places.sql

```diff
@@ -560,20 +562,31 @@ LANGUAGE plpgsql IMMUTABLE STRICT;
  */
 CREATE OR REPLACE FUNCTION ex_TransferDuration(tags jsonb) RETURNS xml AS
 $$
-SELECT
-  CASE
-    WHEN $1->>'duration' IS NOT NULL THEN
+DECLARE
+  duration interval;
+BEGIN
+  SELECT
+    CASE
+      WHEN $1->>'duration' IS NOT NULL THEN
+        duration_to_xsd_duration($1->>'duration')
+      ELSE NULL
+    END
+  INTO duration;
```

Comment:
Minor stuff:
- don't mix SQL and plpgsql if possible. The `CASE WHEN` is SQL while `IF RETURN` etc. is plpgsql
- the part can be simplified to
```
duration := duration_to_xsd_duration($1->>'duration');
```
because passing `NULL` to `duration_to_xsd_duration` will immedtialy return `NULL` because the function is defined with the `STRICT` parameter.

Reply:
Thank you for this hint :)
[37c7397](https://github.com/OPENER-next/osm2vdv462/pull/15/commits/37c7397c914f34945d1fde333a3cb742d083613e)